### PR TITLE
Dubnext all the things

### PR DESCRIPTION
**Pull request by keirog**

### CODE REVIEW

**Comment by debugwand** [comment]
👍

File: server/controllers/alerts/read.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ function renderNewAlertsPage(req, res, next) {
 		//TODO: find a way to do this in a less smelly way
 		const rssPref = onOffPrefsModel.find((pref) => pref.preferenceName === 'publish-rss-feeds');
 		if (rssPref) {
-			rssPref.info = `<a href="https://next.ft.com/myft/following/${userId}.rss" data-trackable="myft-rss">Your RSS feed link</a>`;
+			rssPref.info = `<a href="https://www.ft.com/myft/following/${userId}.rss" data-trackable="myft-rss">Your RSS feed link</a>`;
```
Comment (matthew-andrews) [inline server/controllers/alerts/read.js]: aha be careful.  this might not work.

File: server/lib/fetch-lists.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ function fetchLists(userId, data) {
 							.filter(list => !list.isRedirect)
 							.map(list => {
 								return Object.assign(list, {
-									publicUrl: `https://next.ft.com/myft/list/${list.uuid}`,
+									publicUrl: `https://www.ft.com/myft/list/${list.uuid}`,
```
Comment (matthew-andrews) [inline server/lib/fetch-lists.js]: ditto this

File: views/partials/product-tour-slices/clippings.html [hunk 3]
```diff
@@ -35,7 +35,7 @@
 			</div>
 			<div data-o-grid-colspan="12" class="myft-tour__buttons">
 				{{#if isNextUser}}
-				<a href="{{clippingsLinks.optIn}}" class="myft-tour__button myft-tour__buttons-link" data-trackable="next-user">Return to next.ft.com</a>
+				<a href="{{clippingsLinks.optIn}}" class="myft-tour__button myft-tour__buttons-link" data-trackable="next-user">Return to FT.com</a>
```
Comment (matthew-andrews) [inline views/partials/product-tour-slices/clippings.html]: return the new FT.com?

**Comment by matthew-andrews** [comment]
we might need to have a chat 😁

**Comment by keirog** [comment]
😬  Oops, ok

**Comment by keirog** [comment]
Ohhhh I get it. Jumped the gun a bit

**Comment by keirog** [comment]
(For those following, public *my*FT URLs won’t work as `www.` for folks who aren’t opted into Next/Dubnext)

**Comment by matthew-andrews** [comment]
^^ important as RSS readers are almost certainly **not** opted in

**Pull request merged**